### PR TITLE
Add v4 upgrade guide

### PR DESCRIPTION
**Pull request by rmccue**

### CODE REVIEW

File: other-docs/guides/upgrading/v4.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+The latest CMS module brings with it WordPress version 5.4. Some of the highlights are:
+
+- New [Social Icons and Buttons blocks](https://make.wordpress.org/core/2020/02/27/new-or-updated-blocks-in-wordpress-5-4/)
+- TikTok embeds in the block editor
+
+You can [find the full WordPress 5.4 release notes here](https://wordpress.org/news/2020/03/adderley/), and the developer [field guide here](https://make.wordpress.org/core/2020/03/03/wordpress-5-4-field-guide/).
+
+Note that in line with our development and user experience philosophies, some changes in the CMS module may differ from WordPress changes. Notably in Altis v4, the [fullscreen editor mode](https://make.wordpress.org/core/2020/03/03/fullscreen-mode-enabled-by-default-in-the-editor/) is not enabled by default.
+
```
Comment (roborourke) [inline other-docs/guides/upgrading/v4.md:75]: Do we want to mention the new advanced search option and improved japanese and multi language support? It should at least be covered in the section below I think.

File: other-docs/guides/upgrading/v4.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+### Environment Indicators and Developer Tools Improvements
+
+The current environment you are on (local, development, or staging) is now displayed in the top right corner of the admin screen alongside the Altis logo. This indicator is only displayed on non-production environments.
+
+Additionally, you can now jump directly to the Altis Dashboard page for your current environment, easing the experience of viewing logs, deploying, and debugging requests. A new Open Support Ticket allows you to quickly open tickets for your current environment as well.
+
+The developer tools now display your active Altis configuration, as well as information about Elasticsearch and ElasticPress queries used while rendering the page.
+
+## Local Environment Improvements
```
Comment (roborourke) [inline other-docs/guides/upgrading/v4.md:87]: There's some additional good stuff covered here too https://dev.hmn.md/2020/03/31/altis-v4-sneak-peek/

- Local server db commands
- Local server import uploads command

File: other-docs/guides/upgrading/v4.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+- New [Social Icons and Buttons blocks](https://make.wordpress.org/core/2020/02/27/new-or-updated-blocks-in-wordpress-5-4/)
+- TikTok embeds in the block editor
+
+You can [find the full WordPress 5.4 release notes here](https://wordpress.org/news/2020/03/adderley/), and the developer [field guide here](https://make.wordpress.org/core/2020/03/03/wordpress-5-4-field-guide/).
+
+Note that in line with our development and user experience philosophies, some changes in the CMS module may differ from WordPress changes. Notably in Altis v4, the [fullscreen editor mode](https://make.wordpress.org/core/2020/03/03/fullscreen-mode-enabled-by-default-in-the-editor/) is not enabled by default.
+
+
+## Other Developer Features and APIs
```
Comment (roborourke) [inline other-docs/guides/upgrading/v4.md:77]: The security module now has PHP Basic Auth and limit login attempts features.

Comment (roborourke) [reply other-docs/guides/upgrading/v4.md:77]: Search now has the advanced mode, strict/loose matching option and better multi language support.

File: other-docs/guides/upgrading/v4.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+Lastly you will need to update your database tables using the CLI:
+
+- On [Altis Dashboard](https://dashboard.altis-dxp.com/) find the stack you have deployed to and run `core update-db --network` (`wp` is automatically prepended) in the WP CLI tab
+- For Local Chassis run `composer chassis exec -- wp core update-db --network`
+- For Local Server run `composer local-server cli -- core update-db --network`
+
+If you use Local Chassis you will need to also update Chassis and its extensions by running `composer chassis upgrade`.
+
+
+## Breaking Changes
```
Comment (roborourke) [inline other-docs/guides/upgrading/v4.md:36]: The search module updates will require a reindex of the content.

File: other-docs/guides/upgrading/v4.md (hunk 1 above)
Comment (rmccue) [reply other-docs/guides/upgrading/v4.md:75]: Did we backport that to v3 in the end?

**Pull request merged**